lib/web/apiserver.go

```diff
@@ -1425,6 +1425,19 @@ func (h *Handler) siteNodeConnect(
 	log.Debugf("[WEB] new terminal request for ns=%s, server=%s, login=%s, sid=%s",
 		req.Namespace, req.Server, req.Login, req.SessionID)
 
+	authAccessPoint, err := site.CachingAccessPoint()
+	if err != nil {
+		log.Debugf("[WEB] Unable to get auth access point: %v.", err)
```

Comment:
No need to use `[WEB]` prefix here (and below) anymore. It should already be prefixed for you by the logger.

Reply:
there was no global logger defined for WEB, so I made a issue to work on it later: https://github.com/gravitational/teleport/issues/4110 (since this PR was already approved)